src/Compilers/CSharp/Test/Syntax/Syntax/SyntaxNormalizerTests.cs

```diff
@@ -91,17 +64,7 @@ public void TestNormalizeSwitchRecPattern()
 		{ } => ""not null"",
 		null => ""null"",
 };",
-                @"var x = (object)1 switch
-{
-  int { } => ""two"",
-  { } t when t.GetHashCode() == 42 => ""42"",
-  System.ValueTuple<int, int> (1, _) { Item2: > 2 and < 20 } => ""tuple.Item2 < 20"",
-  System.ValueTuple<int, int> (1, _) { Item2: >= 100 } greater => greater.ToString(),
-  System.ValueType { } => ""not null value"",
-  object { } i when i is not 42 => ""not 42"",
-  { } => ""not null"",
-  null => ""null"",
-};"
+                "var x = (object)1 switch\r\n{\r\n  int { } => \"two\",\r\n  { } t when t.GetHashCode() == 42 => \"42\",\r\n  System.ValueTuple<int, int> (1, _) { Item2: > 2 and < 20 } => \"tuple.Item2 < 20\",\r\n  System.ValueTuple<int, int> (1, _) { Item2: >= 100 } greater => greater.ToString(),\r\n  System.ValueType { } => \"not null value\",\r\n  object { } i when i is not 42 => \"not 42\",\r\n  { } => \"not null\",\r\n  null => \"null\",\r\n};"
```

Comment:
I personally don't see this form readable. Keeping it as verbatim string with NormalizeLineEndings called improves its readability a lot. But let the team decide.

Reply:
Yes, I prefer verbatim strings, too. But on the Ubuntu machine it causes errors...

Reply:
@bernd5 It shouldn't cause any problems if you call `NormalizeLineEndings` on the "expected" argument. The failures you saw in [this run](https://github.com/dotnet/roslyn/runs/2370918006) are unrelated. They are known flakiness issues.

Reply:
But shouldn't it work without NormalizeLineEndings()?

Reply:
@bernd5 The verbatim string seems to have different line endings on Windows vs Linux builds. I don't know the reason unfortunately.